### ISSUE TITLE
Menu Component missing from v1.2.2

### ISSUE THREAD

**Issue by andrevv11** [algolia/vue-instantsearch#346]
Bug Report

Docs here: https://community.algolia.com/vue-instantsearch/components/menu.html
This component seems to be missing from the dist.

Codepen: https://codepen.io/anon/pen/MEdKEW

`<ais-menu></ais-menu>` 

```
[Vue warn]: Unknown custom element: <ais-menu> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

(found in <Root>)
```

**Bug: What is the current behavior?**
The component doesn't appear to be in the build.

**Bug: What is the proposed solution?**
Add menu to the build script?

**What is the version you are using? Always use the latest one before opening a bug issue.**
1.2.2

**Comment by Haroenv** [comment]
You're right, it seems to be missing from the version on npm, but not in the source code even though there have been versions since that PR was merged 🤔

**Comment by Haroenv** [comment]
Oh, I was reading the history wrong, Menu was simply merged _after_ 1.2.2, and hasn't been released yet.

**Comment by andrevv11** [comment]
Thanks - Do you know when it will be released or is there a way for me to use this component in an application currently?

**Comment by rayrutjes** [comment]
Hi @andrevv11 ,

The component is not released yet because we still wanted to investigate a weird behaviour: https://github.com/algolia/vue-instantsearch/pull/341

cc @iam4x

**Comment by rayrutjes** [comment]
It's now released in 1.3.0 🎉
